### ISSUE TITLE
Unexpected horizontal scrolling on docs pages

### ISSUE THREAD

**Issue by twmb** [redpanda-data/documentation#1275]
### Describe the Issue

The existing docs pages has a bit of horizontal scrolling, and I'm not sure that's intentional / would expect it not to be.

Page I'm running into the problem on: https://docs.redpanda.com/docs/beta/manage/security/authentication/#enable-kerberos

<img width="1435" alt="Screenshot 2023-02-21 at 8 45 01 PM" src="https://user-images.githubusercontent.com/1283109/220516589-49974995-0c24-4a22-9fc6-2e8ee9cda1ab.png">

The screenshot above is to the right _edge_ of my screen -- yet the on-page nav is cut off. I have to scroll to the right to see the rest of the page navigation.

<img width="329" alt="Screenshot 2023-02-21 at 8 47 41 PM" src="https://user-images.githubusercontent.com/1283109/220516725-0ee4f8e6-1cf1-4681-a359-1204c1ca2058.png">

**Comment by Feediver1** [comment]
I could not duplicate looking at 23.1. Please investigate @Deflaimun 

@twmb We are wondering if perhaps this was specific to our recent Beta version--can you reproduce when looking at 23.1?

**Comment by twmb** [comment]
Yes -- just checked on https://docs.redpanda.com/docs/manage/security/authentication/#enable-kerberos
<img width="1440" alt="Screenshot 2023-03-02 at 12 00 03 AM" src="https://user-images.githubusercontent.com/1283109/222354509-e0ec3e27-3e16-4ec0-9d2e-ea911d36c5f6.png">

This is a screen shot of one end of my screen to the other -- the right is cut off.

**Comment by JakeSCahill** [comment]
@Deflaimun was this one fixed? I can't reproduce

**Comment by Deflaimun** [comment]
This was fixed before. @twmb can you confirm? If it's happening, please reopen the issue.
